server/auth/registrar.go

```diff
@@ -123,9 +123,8 @@ func (auth *AuthManager) handlePreValidateBond(conn comms.Link, msg *msgjson.Mes
 		AssetID:   assetID,
 		Amount:    uint64(amt),
 		Expiry:    uint64(expireTime.Unix()),
-		BondID:    bondCoinID,
 	}
-	auth.Sign(preBondRes)
+	preBondRes.SetSig(auth.SignMsg(append(preBondRes.Serialize(), preBond.RawTx...)))
```

Comment:
Just linking back to our previous discussion of the options a https://github.com/decred/dcrdex/pull/2196#discussion_r1122413838
The approach implicitly hashes the RawTx but the signature's message is a little non-intuitive, whereas the suggestion from @martonp in the discussion with to put an explicit hash of this sent tx in the response payload.  I preferred to omit it from the payload, but wanted to doc here.